AppTaskQueue/test/helpers/helpers/datastore_v3_pb2.py

```diff
@@ -0,0 +1,3873 @@
+# Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
I don't think we should include machine-generated code in the repo if we can avoid it.